src/components/Favourites.js

```diff
@@ -12,8 +13,8 @@ export default function Favourites() {
       <div className="Favourites-saved">
         <h2>Saved Photos</h2>
         <div className="Favourites-photos">
-          <img src="http://via.placeholder.com/80x80" />
-          <img src="http://via.placeholder.com/80x80" />
+          <img src={props.savedPhotos[0]}/>
+          <img src={props.savedPhotos[1]} />
```

Comment:
@SenamiAhifon  what if my savePhotos have more than 2 pictures? you should have a general solution here, this code is just showing the first 2 photos.